### PR TITLE
Add coveragerc in working dir

### PR DESCRIPTION
**Pull request by yvaucher**
Filter `__manifest__.py` files and content of `tests` folder

### CODE REVIEW

File: 10.0/Dockerfile [hunk 1]
```diff
@@ -47,6 +47,8 @@ COPY ./templates /templates
 COPY ./before-migrate-entrypoint.d /before-migrate-entrypoint.d
 COPY ./start-entrypoint.d /start-entrypoint.d
 COPY ./MANIFEST.in /odoo
+# Place coveragerc in workdir where coverage will be launched from
+COPY ./.coveragerc /
```
Comment (simahawk) [inline 10.0/Dockerfile]: so common files are taken automatically I guess?

Comment (simahawk) [reply 10.0/Dockerfile]: thanks for the pointer :)

**Comment by simahawk** [comment]
```
Step 17/22 : COPY ./.coveragerc /
COPY failed: stat /var/lib/docker/tmp/docker-builder782814251/.coveragerc: no such file or directory
Cleaning up temporary directory...
make: *** [build] Error 1
```

File: 10.0/Dockerfile (hunk 1 above)
Comment (guewen) [reply 10.0/Dockerfile]: The `cp` in `build.sh` must be changed to `cp -r common/{.*,*} ${TMP}` to include the hidden files

Comment (yvaucher) [reply 10.0/Dockerfile]: good catch!

**Comment by simahawk** [comment]
still failing
```
0.02s$ make VERSION=$VERSION build
bash build.sh
Working in /tmp/tmp.bpWHDTACtw
cp: will not create hard link ‘/tmp/tmp.bpWHDTACtw/common’ to directory ‘/tmp/tmp.bpWHDTACtw/.’
Cleaning up temporary directory...
make: *** [build] Error 1
```

**Comment by yvaucher** [comment]
Funny, `cp -r .*` also takes `../*` into account.

I might just want to do this in 2 operations.

**Comment by yvaucher** [comment]
Both of those works, the second seems clearer to me:

```
cp -r src/. dest
cp -rT src/ dest/
```

**Comment by simahawk** [comment]
@yvaucher give it another try.... :rofl:

**Comment by simahawk** [comment]
seems to work finally! tnx @yvaucher :)

**Comment by yvaucher** [comment]
Squashed commits

**Pull request merged**